src/issn.rs

```diff
@@ -1,3 +1,59 @@
+use crate::traits::{Valid, Normalize};
+
+pub struct ISSN {
+    pub identifier: String,
+}
+
+pub fn build_issn(identifier: String) -> ISSN {
+    ISSN {
+        identifier,
+    }
+}
```

Comment:
Curious if this might provide a nicer API (e.g. `ISSN::new("1234-5678").normalize`)

```suggestion
pub fn new(identifier: &str) -> ISSN {
    ISSN {
        identifier.to_string(),
    }
}
```

Reply:
Could probably overload the method too to accept a string slice and convert it to a string to clean up the test code.

Reply:
That's what I was thinking with the `impl Into<String>` and `.into()`, so that it could accept a string or string slice.